internal/gcloud/cloudrunservice.go

```diff
@@ -0,0 +1,89 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package gcloud
+
+import (
+	"crypto/rand"
+	"encoding/hex"
+	"github.com/GoogleCloudPlatform/serverless-sample-tester/internal/util"
+	"strings"
+	"unicode"
+)
+
+// Maximum length of a Cloud Run Service name.
+const maxCloudRunServiceNameLen = 53
+
+// Length of the random suffix at the end of the Cloud Run Service name.
+const cloudRunServiceNameRandSuffixLen = 10
+
+// CloudRunService represents a Cloud Run service and stores its parameters.
+type CloudRunService struct {
+	Name string
+	url  string
+}
+
+// Delete calls the external gcloud SDK and deletes the Cloud Run Service associated with the current cloudRunService.
+func (s CloudRunService) Delete() (err error) {
```

Comment:
Just a note, you don't _have_ to use the pattern of naming your return variable-- you can also just return a specific error (or a tuple as the case may be). This is fine but just wanted to make sure you're aware.